packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -15,19 +15,45 @@ import {
   TX_BATCH_SUBMITTER_LOG_TAG,
 } from '..'
 
+const environment = process.env.NODE_ENV
```

Comment:
Usually `NODE_ENV` is intended to be `development`, `production`, or `test`. I would like us to keep with this convention. We may _also_ want to have an additional environment variable for `ETHEREUM_NETWORK_NAME` which has `mainnet`, `kovan`, `goerli`, etc.

Reply:
@snario that makes sense! in terms of using sentry with this, would we label kovan as staging so we can keep them in the same project? https://docs.sentry.io/product/sentry-basics/environments/

Reply:
Goërli = obviously staging Mainnet is production. I think Kovan is some kind of in-between that we need to treat as prod-like since external users are using it, but it's not mainnet... so we are in a unique situation. Your choice as to how we treat that.

Reply:
i ended up putting `network` as the sentry `environment` variable!